### ISSUE TITLE
contour colormap

### ISSUE THREAD

**Issue by jschueller** [openturns/openturns#1695]
the curves have new colors (#1436) but not the contours

see openturns/lib/src/Base/Func/EvaluationImplementation.cxx:
 current.setColor(Contour::ConvertFromHSV((360.0 * i / levels.getDimension()), 1.0, 1.0)); 
same code is used in Distribution, Field, CovModel, BoxCoxFactory

I suggest to take viridis only (matplotlib default linear colormap) from:
https://github.com/yuki-koyama/tinycolormap/blob/master/include/tinycolormap.hpp

**Comment by mbaudin47** [comment]
I fully agree : we already noticed this with @josephmure. We will manage this in a future work on our own. The work should begin soon. Please do not fix that bug, because it would get duplicated.
